patches/@onfido+react-native-sdk+8.3.0.patch

```diff
@@ -1,8 +1,22 @@
 diff --git a/node_modules/@onfido/react-native-sdk/android/build.gradle b/node_modules/@onfido/react-native-sdk/android/build.gradle
-index b4c7106..d5083d3 100644
+index b4c7106..c6efb0f 100644
 --- a/node_modules/@onfido/react-native-sdk/android/build.gradle
 +++ b/node_modules/@onfido/react-native-sdk/android/build.gradle
-@@ -135,9 +135,9 @@ afterEvaluate { project ->
+@@ -84,6 +84,13 @@ android {
+         sourceCompatibility JavaVersion.VERSION_1_8
+         targetCompatibility JavaVersion.VERSION_1_8
+     }
++
++    publishing {
++        singleVariant('release') {
++            withSourcesJar()
++            withJavadocJar()
++        }
++    }
```

Comment:
Is this patch still required in latest version of onfido sdk?
If not, how about upgrading to latest version 10.5.0?

Reply:
From what we can see, the PR fixing it is not merged so most probably the patch is still needed: https://github.com/onfido/react-native-sdk/pull/157

Reply:
<img width="823" alt="patch conflict" src="https://github.com/Expensify/App/assets/108292595/77476a63-ceb7-4fb2-be30-c55004dee364">

There's patch conflict. Removing node_modules will fix it ofc but hope this won't be issue in deploy build

Reply:
If it does, it's not a big deal – we can just delete the `node_modules` cache and start fresh